client/containers/EventsPanelContainer/index.jsx

```diff
@@ -29,9 +29,10 @@ EventsPanel.propTypes = {
 
 const mapStateToProps = (state) => ({
     // selectors.getShowEventDetails is either true, an event id, or null
-    eventToDetail: selectors.getShowEventDetails(state) === true
-        ? DEFAULT_FORM_VALUES
-        : selectors.getEventToBeDetailed(state),
+    eventToDetail: selectors.getShowEventForm(state) ? selectors.getShowEventDetails(state) === true
```

Comment:
I see you are using `eventToDetail` to know which event is asked for deletion ?
I think you could leave `eventToDetail` for what it has been made for, and instead pass the event as a parameter to the action: `dispatch(showModal({ modalType: 'DELETE_EVENT' , modalProps: event }))`